app/src/main/java/com/android/sample/ui/listActivities/LikedActivities.kt

```diff
@@ -80,17 +81,23 @@ fun LikedActivitiesScreen(
           when (uiState) {
             is ListActivitiesViewModel.ActivitiesUiState.Success -> {
               if (profile == null) {
-                Text(
-                    text = "You are not logged in, Login or Register to see your liked activities",
-                    modifier =
-                        Modifier.padding(8.dp)
-                            .align(Alignment.Center)
-                            .testTag("notConnectedPrompt"),
-                    color = MaterialTheme.colorScheme.onSurface)
-                Button(
-                    onClick = { navigationActions.navigateTo(Screen.SIGN_UP) },
-                    modifier = Modifier.testTag("signInButton")) {
-                      Text("Go to Sign In Page")
+                Column(
+                    horizontalAlignment = Alignment.CenterHorizontally,
+                    verticalArrangement = Arrangement.Center,
+                    modifier = Modifier.align(Alignment.Center)) {
+                      Text(
+                          text =
+                              "You are not logged in. Login or Register to see your liked activities.",
+                          modifier = Modifier.padding(bottom = 16.dp).testTag("notConnectedPrompt"),
```

Comment:
We should really spend some time to put all these text inputs in a single organised file. Otherwise, we do not guarantee consistency with the rest of our project. However, since we did not explicitly impose this for our project, it makes no sense for me to force it here. Same story with magic constant values.

Reply:
I agree with you, we should do it